soc/xtensa/nxp_adsp/imx8ulp/Kconfig.defconfig.series

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_NXP_IMX8ULP
+
+config SOC_SERIES
+	string
+	default "imx8ulp"
+
+config SOC_TOOLCHAIN_NAME
+	string
+	default "nxp_imx8ulp_adsp"
+
+config SOC
```

Comment:
This seems to lack NXP HAL binding support. Please fix this as it's required for the switch to the native drivers. Ideally, support for serial interface should be also added as it's going to be required for debugging.

Reply:
@LaurentiuM1234 I agree with you on configs clean up or other unused variables, but for the nxp-hal binding and uart support I believe we can add it later - we have to check what board we can use from hal, what files we need for the dsp, etc.. it might take a while.

Reply:
I agree that this is **very** time consuming, but unless there's a time constraint I would still suggest getting it out of the way now. This is because later on we may have time constraints and this may become a problem. Either way's fine with me I guess